### PR TITLE
[PDF FIXES]

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
Fixes related to (STORY)[https://trello.com/c/jgjkMxFP/87-form-print-outs-not-showing-all-questions-when-printed]

### CODE REVIEW

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 1]
```diff
@@ -180,7 +180,7 @@ def innovation_step1
           required
           context %(
             <p>
-              If you two or more organisations made a significant contribution to the product/service/management approach (e.g. the business marketing a product, providing a service, or using a technology is different from the unit which developed it) and they both achieved commercial success, then you can make a joint entry. Each organisation should submit separate, cross-referenced, entry forms.
+              If two or more organisations made a significant contribution to the product/service/management approach (e.g. the business marketing a product, providing a service, or using a technology is different from the unit which developed it) and they both achieved commercial success, then you can make a joint entry. Each organisation should submit separate, cross-referenced, entry forms.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [396/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step3.rb [hunk 2]
```diff
@@ -98,7 +98,7 @@ def trade_step3
           conditional :financial_year_date_changed, :true
         end
 
-        textarea :trade_excluded_explanation, "In question A12.6 you said you are applying on behalf of a group but are excluding some member(s)'s financial figures. Please explain why." do
+        textarea :trade_excluded_explanation, "In questions A12.6 and A12.7  you said you are applying on behalf of a group but are excluding some member(s)'s financial figures. Please explain why." do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step3.rb]: Line is too long. [201/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 3]
```diff
@@ -189,7 +189,7 @@ def development_step1
           required
           context %(
             <p>
-              If you two or more organisations made a significant contribution to the product/service/management approach, and achieved commercial success, then you should make a joint entry. Each organisation should submit separate, cross-referenced, entry forms.
+              If two or more organisations made a significant contribution to the product/service/management approach, and achieved commercial success, then you should make a joint entry. Each organisation should submit separate, cross-referenced, entry forms.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [260/100]

File: app/pdf_generators/qae_pdf_forms/general/question_pointer.rb [hunk 4]
```diff
@@ -193,21 +201,45 @@ def render_question_without_ref
...
+      question.help.each do |help|
+        h_text = question.escaped_help(help.text)
+        render_context_or_help_block(h_text) if h_text.present?
+      end
+    end
+  end
+
+  def render_question_hints
+    if question.hint.any?
+      question.hint.each_with_index do |help, index|
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/question_pointer.rb]: Unused block argument - `index`. If it's necessary, use `_` or `_index` as an argument name to indicate that it won't be used.

File: app/pdf_generators/qae_pdf_forms/general/question_pointer.rb [hunk 5]
```diff
@@ -193,21 +201,45 @@ def render_question_without_ref
...
-                       height: 6.5.mm
-        form_pdf.render_text context,
-                             style: :bold
-      else
-        form_pdf.render_text context
+      render_context_or_help_block(question.escaped_context)
+    end
+  end
+
+  def render_question_help_note
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/question_pointer.rb]: Assignment Branch Condition size for render_question_help_note is too high. [15.33/15]

File: app/pdf_generators/qae_pdf_forms/general/question_pointer.rb [hunk 6]
```diff
@@ -193,21 +201,45 @@ def render_question_without_ref
...
+    if question.help.any? && form_pdf.form_answer.urn.blank?
+      question.help.each do |help|
+        h_text = question.escaped_help(help.text)
+        render_context_or_help_block(h_text) if h_text.present?
+      end
+    end
+  end
+
+  def render_question_hints
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/question_pointer.rb]: Assignment Branch Condition size for render_question_hints is too high. [20.12/15]

File: app/pdf_generators/qae_pdf_forms/general/question_pointer.rb [hunk 7]
```diff
@@ -193,21 +201,45 @@ def render_question_without_ref
...
+          form_pdf.render_text question.prepared_text(help.title), style: :bold
+        end
+
+        r_text = help.text.squeeze(" ").delete("\n")
+        form_pdf.render_text question.prepared_text(r_text)
       end
     end
   end
 
+  def render_context_or_help_block(context)
```
Comment (matthewford) [inline app/pdf_generators/qae_pdf_forms/general/question_pointer.rb]: Assignment Branch Condition size for render_context_or_help_block is too high. [17.03/15]

File: lib/qae_form_builder/question.rb [hunk 8]
```diff
@@ -236,6 +246,10 @@ def initialize q
       @q = q
     end
 
+    def pdf_title text
```
Comment (matthewford) [inline lib/qae_form_builder/question.rb]: Use def with parentheses when there are parameters.

File: lib/qae_form_builder/question.rb [hunk 9]
```diff
@@ -174,24 +174,34 @@ def required_visible_filled?
     end
 
     def escaped_title
-      if delegate_obj.title.present?
-        title = delegate_obj.title
+      r_title = ''
```
Comment (matthewford) [inline lib/qae_form_builder/question.rb]: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

**Comment by dreamfall** [comment]
:shipit:

**Pull request merged**